test/openapi3/openapi3-test.js

```diff
@@ -68,6 +68,40 @@ describe('input-validation middleware tests', function () {
                     done();
                 });
         });
+        it('validate depending on content-type -- valid dog', function (done) {
```

Comment:
Can you check if these tests are still failing without your changes? Theoretically this should have been handled already at api-schema-builder level, see https://github.com/Zooz/api-schema-builder/pull/25/files

Reply:
Indeed. The first test passes (valid dog) since `foo` is a valid string and conforms to the definition in the `application/json` schema definition, but without my change, the `invalid dog` test fails with:
```
  281 passing (2s)
  1 failing

  1) input-validation middleware tests
       firstError=false
         validate depending on content-type -- invalid dog:
     Uncaught Error: expected 400 "Bad Request", got 200 "OK"
```